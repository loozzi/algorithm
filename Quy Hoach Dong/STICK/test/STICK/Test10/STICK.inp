7

29
93 97
18 99
29 85
66 27
50 86
100 86
56 35
66 18
74 15
34 95
21 78
50 3
2 40
53 49
72 36
40 50
79 93
1 70
13 69
63 11
49 64
35 53
9 82
97 92
97 1
51 36
10 26
65 90
49 51

40
24 56
55 80
7 89
46 78
64 84
58 69
12 7
54 95
59 45
49 73
58 50
81 38
84 47
76 33
53 10
32 20
92 47
62 36
66 3
15 78
49 58
2 31
25 79
25 40
48 87
5 88
80 98
79 3
55 79
59 73
91 16
100 16
6 55
73 21
95 10
4 82
22 58
83 39
34 1
63 95

19
19 6
90 19
67 91
45 5
34 55
41 64
13 72
39 6
57 97
53 94
58 37
38 23
20 39
20 5
14 69
78 36
98 14
85 14
63 29

2
60 2
79 48

6
49 23
25 33
99 78
69 30
49 36
9 55

23
50 89
5 62
8 75
93 82
72 12
83 36
32 96
62 98
39 85
97 73
96 13
77 7
42 24
74 77
70 1
88 28
42 11
37 36
48 57
90 3
6 7
87 90
61 4

32
97 1
44 64
94 54
72 68
72 64
42 70
22 50
16 77
7 52
25 49
17 56
74 96
71 21
12 55
58 65
83 87
78 1
22 8
88 36
12 80
6 95
9 68
49 34
24 6
93 7
33 4
7 67
50 44
40 55
82 76
84 100
2 32
