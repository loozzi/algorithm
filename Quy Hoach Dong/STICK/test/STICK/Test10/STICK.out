9
10
7
1
3
6
10
